lib/galaxy/config/__init__.py

```diff
@@ -624,13 +625,7 @@ def __init__(self, **kwargs):
 
     def _load_schema(self):
         # Schemas are symlinked to the root of the galaxy-app package
```

Comment:
Yes, but should we also move the schemas to lib/galaxy/config, so that they are part of the app package ? That's what's causing the package test failure.

Reply:
yes, I think moving it makes sense.

Reply:
Done in https://github.com/galaxyproject/galaxy/pull/13147/commits/aa07926bb64cd56e218674d0285cf97072d4aa1a

Reply:
Agreed, but should these files in a `schemas` subdirectory, like we have for samples?